library/src/main/java/com/google/android/exoplayer/text/eia608/Eia608Parser.java

```diff
@@ -153,6 +156,12 @@
         continue;
       }
 
+      // Ignore repeated control characters.
+      if (ccData1 == previousCtrl1 && ccData2 == previousCtrl2) {
```

Comment:
I'm not up to speed to the level of detail required to understand your comment yet (sorry :)), but to ask the my question another way, if the spec says to de-duplicate only if ccData1 is in the range [0x10, 0x1F], why would you implement logic to de-duplicate in cases outside of this range?

I'm not sure if you're arguing that (a) you should de-dupe outside of this range too, or (b) it doesn't make a difference if you de-dupe outside of this range too because duplicate operations wont actually do anything. If (b) then it seems best not to de-dupe; it's easier to understand and match onto the spec with it done that way.

Please clarify. Thanks!